test/e2e/setup.sh

```diff
@@ -35,10 +37,11 @@ function create_cluster {
 #
```

Comment:
NIT: The description above this line (GitHub doesn't allow me to comment here) directly refers to setting a variable. Maybe update it so that it reflects that it needs to be EXPORTed.